### PR TITLE
Pre release 0.20.0

### PR DESCRIPTION
**Pull request by prestist**
https://github.com/coreos/coreos-installer/issues/1358

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -4,7 +4,21 @@ nav_order: 8
...
+Minor changes:
+
+
+Internal changes:
+
+
+Packaging changes:
+
+
+## Upcoming coreos-installer 0.20.0 (2023-12-19)
```
Comment (jbtrystram) [inline docs/release-notes.md:21]: > 
```suggestion
## coreos-installer 0.20.0 (2023-12-19)
```

**Review by jbtrystram — CHANGES_REQUESTED** [review]
i think there is one leftover

File: docs/release-notes.md (hunk 1 above)
Comment (prestist) [reply docs/release-notes.md]: Oh wow, thank you for catching that.

**Review by jbtrystram — APPROVED** [review]
lgtm

**Pull request merged**